how_to_use.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: How to use Data Commons
+nav_order: 1
+---
+
+{: .no_toc}
+# How to use Data Commons
+
+* TOC
+{:toc}
+
+## Learn about the data in Data Commons
+
+To find out what data is available in Data Commons, check out the [Statistical Variable Explorer](https://datacommons.org/tools/statvar) and see the [Data sources](/datasets/index.html) pages.
+
+## Issue interactive data queries
+
+For quick analysis, use the search query bar on the home page or any of the visualization tools, such as the [Timeline](https://datacommons.org/tools/visualization#visType=timeline), [Scatter](https://datacommons.org/tools/visualization#visType%3Dscatter), and [Map](https://datacommons.org/tools/visualization#visType%3Dmap) explorers.
```

Comment:
It could be helpful to also link to the homepage.

Also, to me, it reads as if they're also supposed ot use the search query bar on the vis tools. Should it be `... or use any of the ...` or `... or explore using any of the visualization tools ...`

Reply:
Ah, nice example of syntactic ambiguity!  Done.